src/aioquic/quic/recovery.py

```diff
@@ -334,10 +334,7 @@ def _get_loss_space(self) -> Optional[QuicPacketSpace]:
         return loss_space
 
     def _log_metrics_updated(self, log_rtt=False) -> None:
-        data: Dict[str, Any] = {
-            "bytes_in_flight": self._cc.bytes_in_flight,
-            "cwnd": self._cc.congestion_window,
-        }
+        data: Dict[str, Any] = self._cc.get_log_data()
         if self._cc.ssthresh is not None:
```

Comment:
This doesn't look needed, we're already reporting `ssthresh` in `QuicCongestionControl.get_log_data()`